### ISSUE TITLE
My fellow ICE users. PLEASE make the switch to SRM.

### ISSUE THREAD

**Issue by HEspoke** [scottrice/Ice#473]
Hello my Emulation Community,
Just thought I would pop in and check out what's going on with this github. I used to use ICE all the time and had to search the caverns of the internet finding bits of code from forked versions just to hobble along trying to get ICE to work in modern times. PLEASE make the switch to [Steam Rom Manager](https://steamcommunity.com/groups/SteamRomManager). There is a bit of a setup process to get it to work correctly but just like ICE, once you go through the growing pains of setting it up, it's well worth it! The SRM developer is very active on his github and will help with any issues you have. We also set up a discord server that be found [here](https://steamcommunity.com/groups/SteamRomManager) and we will walk you through the process step by step. Please check it out guys. Thanks!

**Comment by oblitum** [comment]
Agreed.

**Comment by Taffer1** [comment]
I'll switch when there is any semblance of ICE's straight forward/simplified configuration.

**Comment by oblitum** [comment]
Well, SRM was pretty simple for me, took me less time than the time I took to give up on ICE. I tried both at around the same time, ICE first, then SRM.

**Comment by HEspoke** [comment]
@Taffer1 SRM might not be for your then. I don't see the dev ever ditching the UI and going to a .txt format for users to manually edit every option. Sorry

**Comment by Taffer1** [comment]
@HEspoke That's a very cute attempt at misinterpreting my statement. It has nothing at all to do with gui vs text, and everything to do with clear concise instructions/useful documentation. I'm not joining a discord server just to learn the basics of using a supposedly simply utility. I want to be able to select my emulator location, select my rom location, and presto!

**Comment by HEspoke** [comment]
@Taffer1 I don't remember ICE being as easy as selecting a emulator location, selecting a rom location and presto....., but if it works for you then I am very happy for you. I would rather people keep using what works for them when it comes to emulation simplicity then using SRM and think it's overly complicated and frustrating. If ICE is a better solution for you then please continue using it =)

**Comment by minexew** [comment]
Ugh, there's no reason why such trivial application should be a 70MB download. I'm not going to recommend bloated software.

**Comment by oblitum** [comment]
> Ugh, there's no reason why such trivial application should be a 70MB download. I'm not going to recommend bloated software.

Right, but it's better than nothing, or better than non-working software ☺️. I've tried both and stick SRM, it's simple and it works. I couldn't make ICE to work.

**Comment by NOFUNEVER** [comment]
I can't recommend SRM to anyone because then i'd have to explain how to use it afterword. A quality of life tool shouldn't make work for the person recommending it ;) Just because I/you can figure it out doesn't mean it's not needlessly convoluted and beyond the patience of most people looking for a tool like this.

**Comment by Teutonic84** [comment]
There's no options for automation with SRM currently. With ICE there is, so that's why i'm still using it. With ICE I can run a batch file that will import any new ROMS I add to a specific folder and bam it's in steam. I can't do that with SRM. I'd have to open up the application, then scan for new ROMS and then tell it to import to steam. ICE is just a much simpler and straight forward application from a command line junkies perspective.
